homeassistant/components/mqtt/client.py

```diff
@@ -268,6 +268,21 @@ def remove() -> None:
     return remove
 
 
+def _is_simple_subscription(topic: str) -> bool:
+    """Return if a topic is a simple subscription."""
+    return not any(
+        (
+            "+" in topic,
+            "#" in topic,
+            "$" in topic,
+            "{" in topic,
+            "}" in topic,
+            "[" in topic,
+            "]" in topic,
+        )
+    )
```

Comment:
I think this could be more efficient

Reply:
That will be fast enough I think

Reply:
Pushed a change.  Looks like we only care about `+` or `#`